gap/ArangoDB.gi

```diff
@@ -165,13 +165,17 @@ InstallGlobalFunction( GapToJsonStringForArangoDB,
         c := c + Length( Positions( chunk, '\"' ) );
         
         if IsEvenInt( c ) then
-            Error( "splitting the input line for arangosh in the middle of a non-string is not supported yet\n" );
+            k := PositionNthOccurrence(Reversed(chunk), ',', 1);
+            if k = fail then
+                Error( "splitting the input line for arangosh in the middle of a non-string is not supported yet\n" );
```

Comment:
Maybe one can then split the line at a white space. What do you think?

Reply:
That's a good idea! Or at a ':' if there is no whitespace to split at. I'll implement this tomorrow.